### PR TITLE
Update Readme with Usage Documentation

### PR DESCRIPTION
**Pull request by asecondwill**
Some docs to help users get going.

### CODE REVIEW

**Comment by aldesantis** [comment]
@asecondwill thanks! Just one thing: could you format the code with syntax highlighting? The syntax for doing that is the following:

    ```ruby
    put code here...
    ```

**Comment by asecondwill** [comment]
ok, that should be done

File: README.md [hunk 1]
```diff
@@ -21,6 +21,62 @@ bundle
...
+
+Add an entry into your Translations 
+```yaml
+# config/locales/en.yml
+en:
+  hello: "Hello world"
+  spree:
+    my_report: My Report
+    my_report_description: A New Report
+````
```
Comment (aldesantis) [inline README.md:77]: There's an extra backtick here, it should be three.

File: README.md [hunk 2]
```diff
@@ -21,6 +21,62 @@ bundle
...
+
```
Comment (aldesantis) [inline README.md:78]: Extra blank line here.

File: README.md [hunk 3]
```diff
@@ -21,6 +21,62 @@ bundle
...
+          post :my_report
+        end
+      end
+    end
+  end
+  mount Spree::Core::Engine, at: '/'
+````
+Create a template in app/views/spree/admin/reports/my_report.html.erb
+
+Add an entry into your Translations
```
Comment (aldesantis) [inline README.md:69]: This should be "Add an entry to your translations". 🙂

**Review by aldesantis — CHANGES_REQUESTED** [review]
Thanks! Left a few more comments on things I had missed in my first pass. Good to go once fixed!

**Pull request closed without merging**